### ISSUE TITLE
[Bug]: Crashes when trying to download certain episodes

### ISSUE THREAD

**Issue by flynnguy** [advplyr/audiobookshelf#1893]
### Describe the issue

When downloading certain episodes, it crashes with:
```
/server/libs/njodb/index.js:103
        throw error;
        ^

Error: spawn E2BIG
    at ChildProcess.spawn (node:internal/child_process:420:11)
    at spawn (node:child_process:733:9)
    at /server/libs/fluentFfmpeg/processor.js:148:24
    at FfmpegCommand.proto._getFfmpegPath (/server/libs/fluentFfmpeg/capabilities.js:90:14)
    at FfmpegCommand.proto._spawnFfmpeg (/server/libs/fluentFfmpeg/processor.js:128:10)
    at /server/libs/fluentFfmpeg/processor.js:433:14
    at t (/server/libs/async/index.js:6:1639)
    at a (/server/libs/async/index.js:6:18659)
    at /server/libs/async/index.js:6:2258
    at /server/libs/fluentFfmpeg/processor.js:359:11 {
  errno: -7,
  code: 'E2BIG',
  syscall: 'spawn'
}
```

### Steps to reproduce the issue

1. Add the podcast https://500songs.com/feed/podcast
2. Try and download any one of the following episodes:
- Episode 150: “All You Need is Love” by the Beatles
- Episode 164: “White Light/White Heat” by the Velvet Underground
- Episode 165: “Dark Star” by the Grateful Dead

### Audiobookshelf version

v2.2.23

### How are you running audiobookshelf?

Docker

**Comment by advplyr** [comment]
This podcast has episode descriptions that are over 100k characters. This was crashing when embedding the description in the audio file. I updated this to trim the description for the embed

**Comment by advplyr** [comment]
Fixed in v2.3.0
